schema/xsd/direct.xsd

```diff
@@ -59,6 +59,7 @@
                 </xs:element>
             </xs:choice>
             <xs:element name="signature-type" type="signature-type" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="on-behalf-of" type="signing-on-behalf-of" minOccurs="0" maxOccurs="1"/>
```

Comment:
Jeg tror jeg synes det høres litt guffent ut å ha ulike steder standardverdiene er definert. Hvis den må være definert i koden for asynkrone oppdrag er jeg fristet til å foretrekke at standardverdien for synkrone oppdrag også er definert der.